modules/upgrade/pages/proxy-upgrade.adoc

```diff
@@ -0,0 +1,50 @@
+[[update.suse.manager.proxy]]
+= Update {productname} Proxy
+
+{productname} Proxies are managed in the same way as clients.
+Maintenance updates (MU) can be installed on a {productname} Proxy in the same way as other clients.
+Some updates will require a restart of the proxy service.
```

Comment:
If we are talking in the context of a MU for the SUSE Manager Proxy product... it will **always** require a restart. Not sure how can express this.

Reply:
```suggestion
MU updates require a restart of the proxy service.
```